client/my-sites/activity/filterbar/type-selector/type-selector.jsx

```diff
@@ -190,15 +196,6 @@ export class TypeSelector extends Component {
 
 	isSelected = ( key ) => this.getSelectedCheckboxes().includes( key );
 
-	handleButtonClick = () => {
-		const { isVisible, onButtonClick } = this.props;
-
-		if ( isVisible ) {
-			this.handleClose();
```

Comment:
This was causing the popover to not close since we are toggling it again in onButtonClick